docs/competition/team-support/battery-charging.md

```diff
@@ -26,7 +26,7 @@ The biggest risk of damage or injury is from the banana plugs coming unplugged f
 
 The batteries themselves have many built in safety features and, unless severely abused, are completely safe. All batteries undergoing charging must be placed inside a fireproof charging bag. In the extremely unlikely event of a fire due to a battery DO NOT attempt to tackle the fire - the batteries contain a flammable metal (Lithium) which cannot be safely tackled with most fire extinguishers.
 
-If at any point you have any doubts about safety, immediately summon the Competition Team Support Coordinator (Rich Barlow) and the Health & Safety Coordinator (Scarzy) via [Helpdesk](/competition/team-support/helpdesk).
+If at any point you have any doubts about safety, immediately summon the Health & Safety Coordinator (Scarzy) via [Helpdesk](/competition/team-support/helpdesk).
```

Comment:
We should remove names from this so that the documentation can be used as-is for other years.

Reply:
Fixed in 9bca408